generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -85,8 +85,46 @@ class WithFireSimDesignTweaks extends Config(
 
 // Tweaks to modify target clock frequencies / crossings to firesim defaults
 class WithFireSimDefaultFrequencyTweaks extends Config(
-  // Optional*: Removing this will require adjusting the UART baud rate and
-  // potential target-software changes to properly capture UART output
+  // Optional: This sets the default frequency for all buses in the system to 2 GHz
+  // (since unspecified bus frequencies will use the pbus frequency)
+  new chipyard.config.WithPeripheryBusFrequency(2000.0) ++
```

Comment:
Why do we use pbus frequency as default again? @davidbiancolin

Seems more intuitive to use sbus frequency as default, set that and pbus frequency to 1000, and set tile frequency to 2000.

Reply:
PBUS was chosen because at the time it was the only frequency being set.

Reply:
Can we make the sbus the default then now, and always set the sbus frequency?
Its pretty unintuitive that PBus frequency ends up setting tile frequency, since the tiles aren't even directly adjacent to the pbus.